src/normalize/entities.ts

```diff
@@ -320,6 +324,14 @@ export const FreeBusy = new Entity({
 	n: ['nodata', FreeBusyInstance]
 });
 
+export const WorkingHours = new Entity({
```

Comment:
is this different then FreeBusy ?

Reply:
no its not but if we use the same type and input for two apis, then the lastes api overrides the results of the previous api. so had to create all the inputs and types for working hours diffrently

Reply:
done, no need of new entity